app/views/directives/notifications/notification-counter.html

```diff
@@ -0,0 +1,13 @@
+<a
+  ng-if="!$ctrl.hide"
+  href=""
+  ng-click="$ctrl.onClick()"
+  class="nav-item-iconic relative">
+  <div
+    ng-if="$ctrl.showNewNotificationIndicator"
+    class="notification-counter notification-counter-offset"
+    row main-axis="center" cross-axis="center">
+    <span><!--{{countDisplay}}--></span>
+  </div>
+  <span class="fa fa-bell" aria-hidden="true"></span>
```

Comment:
yeah, we can fix the example, but it is still just example so would need to be done here as well. I tried replacing and the biggest issue is the navbar-alt-pf styling.

Adding the following fixes it up and allows the removal of the other counter settings:

_notifications-counter.html (yeah needs some of the above fixes):

```
<li class="drawer-pf-trigger" ng-if="!$ctrl.hide">
  <a href="" class="nav-item-iconic" ng-click="$ctrl.onClick()">
    <span class="fa fa-bell" title="Notifications" aria-hidden="true"></span>
    <span ng-if="$ctrl.showNewNotificationIndicator" class="badge"> </span>
  </a>
</li>

```

_notifications.less:
```
.navbar-pf-alt .nav .nav-item-iconic .badge {
  font-size: 9px;
  margin: -15px 0 0 -12px;
  min-width: 10px;
  min-height: 10px;
}

```